### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by aleciabenjamin**
[Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2315151/stories/164245802)

As a User
In order to view information about the application
I would like to visit a landing page

In this PR we implemented a  laundryroom controller added a logo and the sign up and login buttons(not functional yet).

### CODE REVIEW

**Review by yanqiuzhang — APPROVED** [review]

**Review by NiciP — APPROVED** [review]

File: app/controllers/laundryrooms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class LaundryroomsController < ApplicationController
```
Comment (faraznaeem) [inline app/controllers/laundryrooms_controller.rb]: Please use proper naming convention
https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e

File: app/views/laundryrooms/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1>Laundryrooms#index</h1>
+<button>Login</button>
+<button>Sign up</button>
+
```
Comment (faraznaeem) [inline app/views/laundryrooms/index.html.erb]: Please us `haml` instead or `erb`
http://haml.info/

File: features/step_definitions/basic_steps.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Given("I visit the {string}") do |page|
+    case page
+    when "landing page"
+        visit root_path
+    else
+        return false
+    end
+end
+  
+
```
Comment (faraznaeem) [inline features/step_definitions/basic_steps.rb]: Please remove extra lines

File: features/visitor_can_visit_a_landing_page.feature [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Feature: Visitor can visit a landing page
+
+   As a User
+   In order to view information about the application
+   I would like to visit a landing page
+
+
+    Scenario:
+    Given I visit the "landing page"
```
Comment (faraznaeem) [inline features/visitor_can_visit_a_landing_page.feature]: Please mind the indentation

File: features/visitor_can_visit_a_landing_page.feature [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Feature: Visitor can visit a landing page
+
+   As a User
+   In order to view information about the application
+   I would like to visit a landing page
+
+
```
Comment (faraznaeem) [inline features/visitor_can_visit_a_landing_page.feature]: Please remove extra lines

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: app/controllers/laundryrooms_controller.rb (hunk 1 above)
Comment (vick3d) [reply app/controllers/laundryrooms_controller.rb]: fixed

File: app/views/laundryrooms/index.html.erb (hunk 2 above)
Comment (vick3d) [reply app/views/laundryrooms/index.html.erb]: fixed

File: features/step_definitions/basic_steps.rb (hunk 3 above)
Comment (vick3d) [reply features/step_definitions/basic_steps.rb]: fixed

File: features/visitor_can_visit_a_landing_page.feature (hunk 4 above)
Comment (vick3d) [reply features/visitor_can_visit_a_landing_page.feature]: fixed

File: features/visitor_can_visit_a_landing_page.feature (hunk 5 above)
Comment (vick3d) [reply features/visitor_can_visit_a_landing_page.feature]: fixed

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**